actions/docker_check_tags.go

```diff
@@ -0,0 +1,46 @@
+package actions
+
+import (
+	"fmt"
+
+	"github.com/codegangsta/cli"
+	"github.com/coreos/go-quay/quay"
+	"github.com/google/go-github/github"
+
+	reg "github.com/deis/deisrel/registry"
+)
+
+// DockerCheckTags is the CLI action for checking that Docker image tags
+// exist in registries we are interested in
+func DockerCheckTags(ghClient *github.Client) func(c *cli.Context) error {
+	return func(c *cli.Context) error {
+		ref := c.GlobalString(RefFlag)
+
+		// setup registries
+		hubClient, err := reg.GetHub("https://index.docker.io/", "", "")
+		if err != nil {
+			return err
+		}
+		hub := reg.NewHubRegistry(hubClient)
+		quay := reg.NewQuayRegistry(quay.NewHTTPClient(nil), reg.NewQuayAuth())
```

Comment:
yea, it'd be better to pass the value to this func since you're only gonna need to look it up once (presumably)